Framework/DataHandling/test/LoadInstrumentTest.h

```diff
@@ -380,55 +379,47 @@ class LoadInstrumentTest : public CxxTest::TestSuite {
     TS_ASSERT(!physicalInst->isParametrized());
 
     // Check the positions of the 6 detectors in the physical instrument
-    const auto &physicalDetectorInfo = ws->detectorInfo();
-    TS_ASSERT_EQUALS(
-        physicalDetectorInfo.position(physicalDetectorInfo.indexOf(1000)),
-        V3D(0, 0, 0));
-    TS_ASSERT_EQUALS(
-        physicalDetectorInfo.position(physicalDetectorInfo.indexOf(1001)),
-        V3D(0, 1, 0));
-    TS_ASSERT_EQUALS(
-        physicalDetectorInfo.position(physicalDetectorInfo.indexOf(1002)),
-        V3D(1, 0, 0));
-    TS_ASSERT_EQUALS(
-        physicalDetectorInfo.position(physicalDetectorInfo.indexOf(1003)),
-        V3D(1, 1, 0));
-    TS_ASSERT_EQUALS(
-        physicalDetectorInfo.position(physicalDetectorInfo.indexOf(1004)),
-        V3D(2, 0, 0));
-    TS_ASSERT_EQUALS(
-        physicalDetectorInfo.position(physicalDetectorInfo.indexOf(1005)),
-        V3D(2, 1, 0));
+    TS_ASSERT_EQUALS(physicalInst->getDetector(1000)->getPos(), V3D(0, 0, 0));
```

Comment:
I am not sure what to do with occurrences of `physicalInst->getDetector()`, this `physicalInst` is not attached to any workspace (I don't even know what it is and the difference with the "neutronic" instrument). Any ideas?